lib/flight_job/outputs/info_job.rb

```diff
@@ -66,56 +53,55 @@ module Outputs::InfoJob
       <% end -%>
     ERB
 
-    register_attribute(section: :main, header: 'ID') { |j| j.id }
-    register_attribute(section: :main, header: 'Script ID') { |j| j.script_id }
-    register_attribute(section: :main, header: 'Scheduler ID') { |j| j.scheduler_id }
-    register_attribute(section: :main, header: 'State') { |j| j.state }
+    register_attribute(header: 'ID') { |j| j.id }
+    register_attribute(header: 'Script ID') { |j| j.script_id }
+    register_attribute(header: 'Scheduler ID') { |j| j.scheduler_id }
+    register_attribute(header: 'State') { |j| j.state }
 
     # Show a boolean in the "simplified" output, and the exit code in the verbose
-    register_attribute(section: :main, header: 'Submitted', verbose: false) { |j| j.submit_status == 0 }
     # NOTE: There is a rendering issue of integers into the TSV output. Needs investigation
-    register_attribute(section: :main, header: 'Submit Status', verbose: true) { |j| j.submit_status.to_s }
+    register_attribute(header: 'Submitted', verbose: false) { |j| j.submit_status == 0 }
+    register_attribute(header: 'Submit Status', verbose: true) { |j| j.submit_status.to_s }
 
-    register_attribute(section: :main, header: 'Submitted at') do |job, verbose:|
+    register_attribute(header: 'Submitted at') do |job, verbose:|
       if verbose
         job.created_at
       else
         DateTime.rfc3339(job.created_at).strftime('%d/%m/%y %H:%M')
       end
     end
 
-    # NOTE: These could be the predicted times instead of the actual, consider
-    # delineating the two
-    register_attribute(section: :main, header: 'Started at') do |job, verbose:|
-      if job.start_time.nil?
-        nil
-      elsif verbose
-        job.start_time
+    start_header = ->(job, verbose:) do
+      job.start_time || verbose ? 'Started at' : 'Estimated Start'
+    end
+    register_attribute(header: start_header) do |job, verbose:|
+      if job.start_time || verbose
+        job.format_start_time(verbose)
       else
-        DateTime.rfc3339(job.start_time).strftime('%d/%m/%y %H:%M')
+        job.format_estimated_start_time(false)
       end
     end
-    register_attribute(section: :main, header: 'Ended at') do |job, verbose:|
-      if job.end_time.nil?
-        nil
-      elsif verbose
-        job.end_time
+
+    end_header = ->(job, verbose:) do
+      job.end_time || verbose ? 'Ended at' : 'Estimated Finish'
```

Comment:
Let's have this as `Estimated end` to match up more clearly with `Ended at`.

Reply:
90209897cc99b8a7041c24755b5e82ad9ece3e4a